k8s/dns-ssl-kube-lego/README.md

```diff
@@ -0,0 +1,198 @@
+# Endpoints DNS with SSL certificate issued by Let's Encrypt through Kube-Lego
+
+This example demonstrates how to deploy an Endpoints service on GKE with automatic
+provisioning of an SSL certificate for the .cloud.goog service domain.
+
+## Requirements
+
+For this example, you need:
```

Comment:
I'd also mention that the user should update their gcloud installation before continuing, `sudo gcloud components update`.

Reply:
Done